### PR TITLE
refactor(page-header): update page header according to DT spec - INNO-441

### PR DESCRIPTION
**Pull request by emeryro**
Page header updated.
Code cleaned up. There were lots of unused code, maybe there are some other use cases, but I didn't found them in DT documentation.

Breadcrumb and meta components will have to be updated later (I kept them as is for now)

### CODE REVIEW

File: framework/components/ecl-page-headers/_page-headers.scss [hunk 1]
```diff
@@ -1,205 +1,30 @@
...
-  }
-
-  &.btn {
-    padding-right: 2.1em;
-  }
-}
-
-.page-header {
-  margin: 0;
-  padding: 0 0 1.5em;
```
Comment (kalinchernev) [inline framework/components/ecl-page-headers/_page-headers.scss]: Any specific reason why removing this?

**Comment by emeryro** [comment]
@kalinchernev I based my implementation on DT specifications (on their wiki), not on DTT implementation (even if I sticked to the design).
There is no references to background images or other display in the specifications...

File: framework/components/ecl-page-headers/_page-headers.scss (hunk 1 above)
Comment (emeryro) [inline framework/components/ecl-page-headers/_page-headers.scss]: I didn't have any reason to keep it, as it didn't change the display

**Comment by emeryro** [comment]
Just discussed with @degliwe and @yhuard, I will add the other variants

**Comment by emeryro** [comment]
Missing modifiers have been added.
Ready for review

**Comment by emeryro** [comment]
Paddings have been fixed.
@degliwe good for you too?

**Comment by emeryro** [comment]
improve-complete has been fixed

**Pull request merged**